test/javascript/components/student/TracksList.test.js

```diff
@@ -12,9 +14,41 @@ test('disables the selected status filter', () => {
   const { getByText } = render(
     <TracksList
       statusOptions={options}
+      tagOptions={[]}
```

Comment:
We'll be able to catch these errors once we've converted everything to TypeScript!